### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by vicfriedman**
ready for review @jmburges @AnnJohn

### CODE REVIEW

**Comment by AnnJohn** [comment]
Reassigned to @ipc103

File: README.md [hunk 1]
```diff
@@ -182,7 +150,7 @@ null;
 
 undefined;
 ```
-  
+
 ## Return Values
```
Comment (ipc103) [inline README.md]: this section on return values would probably make more sense when learning about functions. ie "Functions have return values the same way that methods do in Ruby. However, unlike in Ruby, you must explicity declare..." 

Should we remove this section and move to a later README?

**Comment by ipc103** [comment]
made a couple of minor edits and left a comment about the return values section. otherwise, good to merge.

File: README.md (hunk 1 above)
Comment (vicfriedman) [inline README.md]: @ipc103 sounds good to me - there is a functions codealong/readme, so let's just make sure its explicitly stated in that readme!

Comment (vicfriedman) [inline README.md]: @ipc @AnnJohn removed return values!

**Pull request merged**